README.md

```diff
@@ -160,3 +160,9 @@ Using dynamic username from job option
        ssh-options="-o ConnectTimeout=5000"/>
 ```
 *a option called `username`  must be added to the job
+
+## Note
+This plugin could present some problems when it tries to copy a file to Windows Server nodes. 
+This is because this plugin uses scp command, and it throws some issues with Windows machines.
```

Comment:
`and it throws some issues with Windows machines` -> this need more clarification like:
SCP command is not present on Windows, making the plugin unable to do ...